consensus/src/dag/dag_handler.rs

```diff
@@ -37,22 +41,44 @@ impl NetworkHandler {
         signer: ValidatorSigner,
         epoch_state: Arc<EpochState>,
         storage: Arc<dyn DAGStorage>,
+        payload_client: Arc<dyn PayloadClient>,
+        _dag_network_sender: Arc<dyn DAGNetworkSender>,
+        rb_network_sender: Arc<dyn RBNetworkSender<DAGMessage>>,
+        time_service: Arc<dyn TimeService>,
+        aptos_time_service: aptos_time_service::TimeService,
```

Comment:
Having two time services is ugly :(

Reply:
Yes, the question is which one is preferred. I see you initially used consensus::TimeService but because RB became its own crate so I had to the aptos_time_service crate. I see some implementation differences between both.

Reply:
I think we should unify on the aptos_time_service, we can merge them on tech debt week